packages/vscode-dependencies-validation/src/autofix/unsupportedFilesWatcher.ts

```diff
@@ -0,0 +1,50 @@
+import type { Uri } from "vscode";
```

Comment:
as mentioned elsewhere in this review, I think we should consider keeping this scenario out-of-scope.
as the edge case which consists of:
- remove exclusion (rm yarn.lock)
- having errors to show.

Seems somewhat rare.

Also:
- someone may just be re-generating their yarn.lock so running npm -i in that case would be a mistake.
- if we implement some periodic (hourly) checks it will still handle the edge case of changing from yarn -> npm

Reply:
Let's talk about it